vault/router_access.go

```diff
@@ -16,6 +18,10 @@ func NewRouterAccess(c *Core) *RouterAccess {
 	return &RouterAccess{c: c}
 }
 
+func (r *RouterAccess) StorageByAPIPath(ctx context.Context, path string) logical.Storage {
```

Comment:
This will be removed if I can move the tune test into `logical_system_test.go`.

Reply:
Removed in https://github.com/hashicorp/vault/pull/24962/commits/9e66566e8a397adf1116dde47288d9c1c28ff612.